rustls/src/client/hs.rs

```diff
@@ -484,8 +488,9 @@ impl ExpectServerHello {
         })
     }
 
-    fn into_expect_tls12_certificate(self) -> NextState {
+    fn into_expect_tls12_certificate(self, negotiated_version: ProtocolVersion,) -> NextState {
```

Comment:
Style nit: let's ditch the trailing comma here?

Reply:
Removed.